### PR TITLE
Create registry_event_to_turn_on_script_execution.yml

### PR DESCRIPTION
**Pull request by ThureinOo**
### Summary of the Pull Request

This PR adds a new registry set rule that triggers when enabling powershell script execution

### Changelog
new: PowerShell Script Execution Policy Enabled
update: Potential PowerShell Execution Policy Tampering

### Example Log Event
N/A

### Fixed Issues
N/A

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

**Comment by nasbench** [comment]
@ThureinOo stop opening and closing PRs :)

**Comment by ThureinOo** [comment]
Thanks @nasbench.

**Comment by nasbench** [comment]
So what's the state of this PR @ThureinOo? If its ready for review please fill the template above and provide a summary and changelog :)

**Comment by ThureinOo** [comment]
I have updated the format.

**Review by nasbench — APPROVED** [review]

File: rules/windows/registry/registry_set/registry_set_powershell_enablescripts_enabled.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+title: PowerShell Script Execution Policy Enabled
+id: 8218c875-90b9-42e2-b60d-0b0069816d10
+related:
+    - id: fad91067-08c5-4d1a-8d8c-d96a21b37814
+      type: derived
+status: experimental
+description: Detecs the enabling of the powershell script execution policy. Once enabled, this policy would allow scripts to be executed.
```
Comment (phantinuss) [inline rules/windows/registry/registry_set/registry_set_powershell_enablescripts_enabled.yml:7]:
```suggestion
description: Detects the enabling of the PowerShell script execution policy. Once enabled, this policy allows scripts to be executed.
```

**Review by phantinuss — APPROVED** [review]

**Pull request merged**